.pre-commit-config.yaml

```diff
@@ -8,13 +8,13 @@ repos:
     rev: 4.0.1
     hooks:
       - id: flake8
-        additional_dependencies: [flake8-bugbear]
+        additional_dependencies:
+          - flake8-bugbear==22.4.25
 
   - repo: https://github.com/PyCQA/isort
     rev: 5.10.1
     hooks:
       - id: isort
-        additional_dependencies: [toml]
```

Comment:
`isort` vendors `tomli`, so this is definitely not needed: https://github.com/PyCQA/isort/tree/380c03b1c40415c806d2244f73793e0c9182ca26/isort/_vendored/tomli